tests/locationConfig/locationConfigTests.json

```diff
@@ -109,5 +109,54 @@
             "bucketMatch": true,
             "azureContainerName": "s3createbucketonfly"
         }
+    },
+    "gcpbackendencryption": {
+        "type": "gcp",
+        "legacyAwsBehavior": true,
+        "details": {
+            "awsEndpoint": "storage.googleapis.com",
+            "bucketName": "zenko-gcp-bucket",
+            "mpuBucketName": "zenko-gcp-mpu",
+            "overflowName": "zenko-gcp-overflow",
+            "bucketMatch": true,
+            "credentialsProfile": "google",
+            "serverSideEncryption": true
```

Comment:
does gcp take a header on an object put to encrypt like aws does?

Reply:
GCP object put does not take a header for SSE; it has SSE on by default: https://cloud.google.com/storage/docs/encryption  
I will remove what isn't necessary.

Reply:
ok, so we shouldn't have the option here. it will just confuse people and won't actually accomplish what they want.